packages/palette/src/palette.story.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import styled from 'react-emotion';
```

Comment:
We should at least upgrade to the latest version of emotion and use that instead. Such that this would be `@emotion/styled`.

Reply:
Whoops, you're right!